demisto_sdk/commands/common/schemas/layoutrule.yml

```
@@ -0,0 +1,50 @@
+type: map
+mapping:
+  rule_id:
+    type: str
+    required: true
+  rule_name:
+    type: str
+    required: true
+  layout_id:
+    type: str
+    required: true
+  fromVersion:
+    type: str
+    required: true
+  toVersion:
+    type: str
+    required: true
```

```suggestion
  toVersion:
    type: str
```
lmk if I'm unaware of something

---

Removed the toVersion arg